README.md

```diff
@@ -72,10 +72,18 @@ The following key features have been introduced in Woof! relative to MBF or WinM
  * All IWAD files since Doom 1.2 are supported, including Doom Shareware, The Ultimate Doom, Doom 2 and Final Doom. Furthermore, the IWAD files shipped with the "Doom 3: BFG Edition" and the ones published by the Freedoom project as well as Chex Quest, HACX and REKKR are supported.
  * UMAPINFO lumps for MASTERLEVELS.WAD, NERVE.WAD, SIGIL_V1_21.WAD, E1M4B.WAD and E1M8B.WAD have been added which are meant to be autoloaded with the corresponding PWAD and change map title, level transitions, par times, music and skies accordingly.
  * The concept of compatibility levels has been added, currently offering "Vanilla", "Boom", "MBF" and "MBF21" (default). The default compatibility level may be changed through the menu and overridden with the `-complevel` parameter, allowing for both numeric and named arguments, or the `COMPLVL` lump. Menu items in the Setup menu that don't apply to the current compatibility level are disabled and grayed out.
- * The project strives for full demo compatibility for all supported complevels.
+ * The project strives for full demo compatibility for all supported complevels. Furthermore, this port offers quality-of-life features like a demo progress bar, demo playback warp and skip and continue demo recording features.
  * The SPECHITS, REJECT and INTERCEPTS overflow emulations have been ported over from Chocolate Doom / PrBoom+, allowing for some more obscure Vanilla demos to keep sync.
 
-# Download
+# Releases
+
+Source code and Windows binaries (32-bit MSVC build for Windows XP and newer, 64-bit MingW-W64 build for Windows 7 and newer) for the latest release can be found on the [Release](https://github.com/fabiangreffrath/woof/releases/latest) page.
```

Comment:
But we ship a 32-bit MSYS build, not MSVC. I was afraid of DLL incompatibility - usually users unpack the new version into their old directories with old 32bit MSYS DLLs. Not sure if this is the issue, maybe we can provide an MSVC build. Everyone downloads the 64-bit version anyway.

Reply:
I'd expect binary releases to be self-contained and if one release contains a bunch of libraries, I wouldn't even expect it to run with the libraries from the previous version.

Reply:
On Windows, everyone just unpacks the new version into the old directory and expects it to work (that's where the saves and config are). I do it too 😄
If the user unpacks the MSVC version into the old 32-bit version of Woof with the MSYS DLLs, it should still work, right? We can risk it, I'm sure most users download the 64-bit version.

Reply:
Yes, let's just upload the same packages for release that we already do for per-commit CI. That's probably less surprise for us. 😁 Otherwise, I could just as well adapt the README. 🤷